### PR TITLE
[PR 2:1] ci: Dependabot for automated PR's on dependencies

### LINKED ISSUE #22: [RQST]ci: Dependabot for automated PR's on dependencies?

### PR DESCRIPTION
**Pull request by Haruyzal**
Resolves #22

### CODE REVIEW

File: docs/user_docs/temp
Comment (lasselundstenjensen) [inline docs/user_docs/temp:1]: Why is it here? Remove?

File: go.mod
Comment (lasselundstenjensen) [inline go.mod:1]: The `go.mod` file is in the root, but is there a reason it is not in the `src` directory? The `src` directory is essentially `package src` now.

**Review by fadur — APPROVED** [review]
Looks good. 
I would however look at the folder structure.
Read up on idiomatic folder/package structure in golang.

**Review by lasselundstenjensen — APPROVED** [review]
Looks good in general.
Consider the comments and make changes or consider adding them as issues for later.

File: docs/user_docs/temp
Comment (Alex-itu) [reply docs/user_docs/temp:1]: We removed it in pull request [30](https://github.com/NovoNordisk-OpenSource/decentralized-tech-radar/pull/30)

File: go.mod
Comment (Alex-itu) [reply go.mod:1]: Let's do a discussion on this, at one of the coming meetings.

**Comment by Haruyzal** [comment]
> Looks good. 
> 
> I would however look at the folder structure.
> 
> Read up on idiomatic folder/package structure in golang.

Please have a look at [PR 2:6](https://github.com/NovoNordisk-OpenSource/decentralized-tech-radar/pull/49) :)

**Pull request merged**